README

```diff
@@ -31,9 +31,23 @@ If the optional -c is given, the board will upon receiving the tilde sequence
 restart the board the given number of times. Each time booting the given
 boot.img.
 
+The optional -s argument can be used to specify that a fifo should be created
```

Comment:
This might change your the interface, but maybe it's better to use `socket(AF_UNIX, SOCK_SEQPACKET, 0)` insead of a FIFO? This way you won't have to worry about the message boundaries.

Reply:
Rely on newline as message boundary means that both on the status-cmd side, and on the client fifo side it's just a stream of bytes. Meaning that both generation and consumption can conveniently be done with a simple script, with out messing with sockets.

Reply:
acl